corehq/apps/userreports/adapter.py

```diff
@@ -82,8 +82,10 @@ def bulk_save(self, docs):
         Evalutes UCR rows for given docs and saves the result in bulk.
         Override this to support bulk. Currently supported in SQL version
         """
+        rows = []
         for doc in docs:
-            self.save(doc)
+            rows.extend(self.get_all_values(doc))
+        self.save_rows(rows)
```

Comment:
I think this should still be on the `SqlAdapter` class as `save_rows` isn't defined on this base class, so this would error if we implemented a different database again.